symfony/client/src/orangehrmAdminPlugin/pages/emailConfiguration/ViewEmailConfiguration.vue

```diff
@@ -0,0 +1,351 @@
+        </oxd-form-actions>
+      </oxd-form>
+    </div>
+  </div>
+</template>
+
+<script>
+import {APIService} from '@orangehrm/core/util/services/api.service';
+import SwitchInput from '@orangehrm/oxd/src/core/components/Input/SwitchInput';
+import { required, validEmailFormat } from '@orangehrm/core/util/validation/rules'
+import { shouldNotExceedCharLength } from '@/core/util/validation/rules'
+
+export default {
+  props: {
+    pathToSendmail: {
+      type: String,
+      required: true,
+    },
+  },
+  setup() {
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      'api/v2/admin/email-configuration',
+    );
+    return {
+      http,
+    };
+  },
+
+  components: {
+    'oxd-switch-input': SwitchInput,
+  },
+
+  data() {
+    return {
+      userSecureConnection: false,
+      sendTestMailEditable: false,
+      isLoading: false,
+      emailConfiguration: {
+        mailType: '',
+        sentAs: '',
+        pathToSendmail: this.pathToSendmail,
+        smtpHost: '',
+        smtpPort: null,
+        smtpUsername: '',
+        smtpPassword: '',
+        smtpAuthType: '',
+        smtpSecurityType: '',
+        testEmailAddress: '',
+      },
+      rules: {
+        mailType: [required, shouldNotExceedCharLength(50)],
+        sentAs: [required, shouldNotExceedCharLength(250), validEmailFormat],
+        smtpHost: [required, shouldNotExceedCharLength(250)],
+        smtpPort: [shouldNotExceedCharLength(10)],
+        smtpUsername: [required, shouldNotExceedCharLength(250)],
+        smtpPassword: [required, shouldNotExceedCharLength(250)],
+        smtpAuthType: [shouldNotExceedCharLength(50)],
+        smtpSecurityType: [shouldNotExceedCharLength(50)],
+        testEmailAddress: [required, shouldNotExceedCharLength(250), validEmailFormat],
+      },
+    };
+  },
+
+  methods: {
+    onSave() {
+      this.isLoading = true;
+      this.http.http
+        .put('api/v2/admin/email-configuration', {
+          mailType: this.emailConfiguration.mailType,
+          sentAs: this.emailConfiguration.sentAs,
+          smtpHost: this.emailConfiguration.smtpHost,
+          smtpPort: parseInt(this.emailConfiguration.smtpPort),
+          smtpUsername:
+            this.emailConfiguration.smtpAuthType === 'login'
+              ? this.emailConfiguration.smtpUsername
+              : '',
+          smtpPassword: this.emailConfiguration.smtpPassword,
+          smtpAuthType: this.emailConfiguration.smtpAuthType,
+          smtpSecurityType: this.userSecureConnection
+            ? this.emailConfiguration.smtpSecurityType
+            : 'none',
+          testEmailAddress: this.emailConfiguration.testEmailAddress,
+        })
+        .then((response) => {
+          const testEmailStatus = response.data.meta?.testEmailStatus;
+          if (testEmailStatus === 1 && this.sendTestMailEditable) {
+            return this.$toast.success({title: 'Success', message: 'Successfully Saved. Test Email Sent'});
```

Comment:
These toast messages are in spec?

Reply:
No, I got them from 4.x

Reply:
Please get confirmations.

Reply:
discussed with kavinda. set them as 2 toast messages:
1: Successfully Saved
2: Test Email Sent/ Not Sent